client/extensions/woocommerce/app/order/order-fulfillment/index.js

```diff
@@ -175,8 +181,8 @@ class OrderFulfillment extends Component {
 		return (
 			labelsEnabled &&
 			hasLabelsPaymentMethod &&
-			this.isAddressValidForLabels( storeAddress ) &&
-			this.isAddressValidForLabels( shipping )
+			this.isAddressValidForLabels( storeAddress, 'origin' ) &&
+			this.isAddressValidForLabels( shipping, 'destination' )
```

Comment:
In the function `isAddressValidForLabels()` we were previously checking whether the address's state was in the accepted list of states ([on line 169](https://github.com/Automattic/wp-calypso/blob/899b90520d24676171f801c83c964bb9650eaf8e/client/extensions/woocommerce/app/order/order-fulfillment/index.js#L169)). Do we no longer care if the destination address's state is in the accepted list of states?

Reply:
No, we no longer care. If the flag is enabled, now the logic is simpler:
* For destination addresses, everything goes.
* For origin addresses, the only restriction is that the country is one of the `USPS_COUNTRIES`.